src/NetTopologySuite/Geometries/Implementation/CoordinateArraySequence.cs

```diff
@@ -119,7 +119,7 @@ public CoordinateArraySequence(CoordinateSequence coordSeq)
         {
             if (coordSeq == null)
             {
-                Coordinates = new Coordinate[0];
+                Coordinates = Array.Empty<Coordinate>();
```

Comment:
:page_facing_up: Same basic note about behavior change as in `MinimumBoundingCircle.cs`